index.js

```diff
@@ -48,4 +97,45 @@ window.onload = document.getElementById('MyForm').onsubmit = calculate_sphere; /
 //Now in this exercise we want you to create 3 buttons, each with a click action that will hide the respective question's answer above. So if you click the "Hide Question 1" button it will hide from the DOM the answer to your first exercise. If you click it again it will show the answer. 
 
 
-//resolve // QUESTION 4 here
+//resolve // QUESTION 4 here 
+questionA = document.getElementById('a-1');
+buttonA = document.getElementById("a-4");            
+buttonOne = document.createElement("BUTTON");
+buttonOne.innerText = 'Hide or Show Answer for Number One';
+buttonA.appendChild(buttonOne);
+buttonOne.addEventListener('click', myFunctionA);
+function myFunctionA() {
+     if (questionA.style.display == "none") {
+       questionA.style.display = "block";
+     } else {
+       questionA.style.display = "none";
+     }}
+  
+     questionB = document.getElementById('a-2');
+     buttonB = document.getElementById("a-4");            
+     buttonTwo = document.createElement("BUTTON");
+     buttonTwo.innerText = 'Hide or Show Answer for Number Two';
+     buttonB.appendChild(buttonTwo);
+     buttonTwo.addEventListener('click', myFunctionB);
+     function myFunctionB() {
+          if (questionB.style.display == "none") {
+            questionB.style.display = "block";
+          } else {
```

Comment:
variables can be declared https://www.w3schools.com/js/js_variables.asp, 

indentations needs to be correct https://www.w3schools.com/js/js_conventions.asp

the code needs to be commented https://www.w3schools.com/js/js_comments.asp

Reply:
Variable names need to be more descriptive

Reply:
myFunctionA , B and C are doing similar job, we can write only one function and reuse it for three buttons